kubernetes/apps/web3/monero/dashboard/resources/dashboard.css

```
@@ -1,630 +1,716 @@
+  --accent-dim: color-mix(in oklch, var(--accent) 14%, transparent);
+  --accent-line: color-mix(in oklch, var(--accent) 35%, transparent);
+  /* Chart.js reads these through getComputedStyle, so they live here too */
+  --chart-fill: color-mix(in oklch, var(--accent) 12%, transparent);
+  --chart-grid: color-mix(in oklch, var(--line) 60%, transparent);
+
+  --positive: oklch(0.76 0.15 155);
+  --positive-line: color-mix(in oklch, var(--positive) 40%, transparent);
+  --negative: oklch(0.68 0.19 25);
+
+  --font-ui:
+    system-ui, -apple-system, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)